pkg/controller/backup_sync_controller.go

```diff
@@ -84,10 +99,13 @@ func (c *backupSyncController) run() {
 	}
 	c.logger.WithField("backupCount", len(backups)).Info("Got backups from object storage")
 
+	cloudBackups := make(map[string]string)
```

Comment:
I was thinking you could use a `sets.String` (from `k8s.io/apimachinery/pkg/util/sets`). We use this throughout the codebase for situations like this.